### ISSUE TITLE
Newlines in Llama 3 End Sequence breaking it

### ISSUE THREAD

**Issue by wereretot** [LostRuins/koboldcpp#799]
Having newlines in the end sequence causes the model to continue as it doesn't put newlines down most of the time, so usually it starts moralizing.

**Comment by wereretot** [comment]
This is happening with 70B IQ_2_XS

**Comment by wereretot** [comment]
Removing the newlines causes the model to stop as expected.

**Comment by LostRuins** [comment]
I will remove the newlines from the llama 3 format in the next version. Seems to be generally negative.

**Comment by wereretot** [comment]
@LostRuins Yeah, even though I'm pretty sure the official format says to do it.

**Comment by LostRuins** [comment]
Hi, Should be fixed in the latest version, turns out this was due to a bad tokenizer merge. Try it again with a freshly reconverted GGUF.
